src/main/java/com/hyune/raider50g/domain/booking/Booking.java

```diff
@@ -0,0 +1,69 @@
+package com.hyune.raider50g.domain.booking;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.hyune.raider50g.common.type.ClassType;
+import com.hyune.raider50g.domain.booking.dto.BookingCommand;
+import java.io.Serializable;
+import java.time.LocalDateTime;
+import javax.persistence.Column;
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@Builder
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@NoArgsConstructor
+@Entity
+public class Booking implements Serializable {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+
+  @Builder.Default
+  @Column(nullable = false)
+  private Boolean cancel = false;
```

Comment:
Wrapper Class를 사용한 이유는요?

Reply:
Booking 은 Entity 이기에 DB 와 직접 연결 됩니다.
그리고 DB 에 연결되는 값은 Wrapper Class 를 사용하는 것이 더 안전하다고 알고 있습니다.

Reply:
네, 저도 그렇게 알고 있고 JPA에서 객체 매핑을 할 때에는 wrapper class를 사용하려고 합니다.
https://stackoverflow.com/questions/7506802/using-wrapper-integer-class-or-int-primitive-in-hibernate-mapping
```
Use wrappers, make your life simple.

Your data model should dictate this. You should be enforcing nullability in the database anyway.

If they are nullable in the database, then use wrappers. If they are not nullable, and you use wrappers, then you'll get an exception if you try and insert a null into the database.

If your data model doesn't dictate it, then go for a convention, use wrappers all of the time. That way people don't have to think, or decide that a value of 0 means null.

I would also query your assertion that it would less performant. Have you measured this? I mean really measured it? When you're talking to a database, there are a lot more considerations than the difference between 16 bits and 32 bits.

Just use the simple, consistent solution. Use wrappers everywhere, unless somebody gives you a very good reason (with accurate measured statistics) to do otherwise.
```